src/App.vue

```diff
@@ -42,6 +43,15 @@ export default {
     ...mapState('aeternity', ['sdk']),
   },
   async created() {
+    window.addEventListener('unhandledrejection', (error) => {
+      if (error !== 200 && this.$route.name !== 'maintenance') {
```

Comment:
Nothing should throw errors with 200 code. Also, I would remove the maintenance page change, because currently, we have no plan to use it. Maybe we can remove it at all?

Reply:
right, thanks